env.example

```diff
@@ -1,4 +1,6 @@
 export LOCATION=eastus
 export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
+export RP_MODE=development
+export SECRETS=secrets.public
```

Comment:
The idea is that you would create your own files, like `env.public` and `env.usgov` that have the value hardcoded. So in `env.public` it would be:
`export SECRETS=secrets.public`
...and in `env.usgov` it would be:
`export SECRETS=secrets.usgov`

And then you `source ./secrets.public`, for example

Not too clever, but being too clever isn't always clever.

Reply:
Do we need to update step 4 [here](https://github.com/Azure/ARO-RP/blob/8801190e8be8540b23c77cff6464bbc1b9b28df4/docs/deploy-development-rp.md#prepare-your-environment) then? Not so much about being clever, but more on proper usage of bash to keep things DRY.

Reply:
I'll update that doc